src/map.cpp

```diff
@@ -686,14 +686,14 @@ bool Map::getPathMatching(const Creature& creature, const Position& targetPos, s
 		return false;
 	}
 
-	static constexpr std::array<std::pair<int, int>, 8> allNeighbors = {
+	const std::array<std::pair<int, int>, 8> allNeighbors = {
```

Comment:
When I said spectacular use of static constexpr, I wasn't being sarcastic, that was a really great usage of it! I'm sorry if there was a miscommunication there as these reviews typically only have comments that are meant for changes, but this was too good to not say something nice about it, that's all.

Reply:
Something changed in the main branch and it really messed up this build lol. Idk what I need to do to fix it.